Scalar.Common/Maintenance/ConfigStep.cs

```diff
@@ -208,25 +209,86 @@ public bool TrySetConfig(out string error)
                 return false;
             }
 
-            return this.ConfigureWatchmanIntegration(out error);
+            this.GetConfigSettings();
```

Comment:
(I want to pin this comment to `TrySetConfig()` but there aren't any deltas near it, so I'm putting it here.)

When is `TrySetConfig()` called?  Is it only during the `scalar clone` and/or registration -- or does it also happen during periodic maintenance?  I'm wondering what happens if we have an existing working directory with Watchman enabled and we now have a version of Git with fsmonitor--daemon available.  (Or I guess vice versa.)
Do we need to explicitly stop Watchman or unregister this working directory from Watchman.  FSMonitor--daemon is implicitly started on demand by a command like `git status`, so we don't have to worry about that part.  But then again, if we switch from the daemon back to watchman, we might need to stop mine and restart and register with watchman.  Maybe that is handled elsewhere.

Reply:
When Scalar upgrades, it runs `scalar run config` on each registered repo. That gives it the ability to "upgrade" config like this as Scalar changes.

Reply:
We should consider what it would take to completely disable watchman. I also thought that maybe it would be good to run a `git status` here after configuring FS Monitor.

Reply:
Yes, running status after configuring would auto-start it and seed it so that the user's first real status would actually benefit.

Reply:
> I also thought that maybe it would be good to run a git status here after configuring FS Monitor.

Could even make it a 'fire and forget' process start for `git status` to avoid slowing the `scalar run config` command acting over multiple repos. The result of the `status` isn't important.